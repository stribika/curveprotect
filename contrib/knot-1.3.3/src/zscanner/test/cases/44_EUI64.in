$ORIGIN	.
$TTL	1

; OK
@	EUI64	00-00-00-00-00-00-00-00		; The simplest case
@	EUI64	FF-FF-FF-FF-FF-FF-FF-FF		; The maximal case
@	EUI64	aa-bb-cc-dd-ee-ff-01-02		; Lower-case
@	EUI64	\# 8 0000000000000000		; Hexadecimal rdata
@	TYPE109	\# 8 0000000000000000		; TYPE + Hexadecimal rdata
@	TYPE109	00-00-00-00-00-00-00-00		; TYPE
@	eui64	00-00-00-00-00-00-00-00		; Type in lower-case

; KO
@	EUI64
@	EUI64					; Empty rdata
@	EUI64	\# 0				; Hex empty rdata
@	EUI64	\#				; Missing hex length
@	EUI64	00-00-00-00-00-00-00		; Too few hex pairs
@	EUI64	00-00-00-00-00-00-00-00-00	; Too many hex pairs
@	EUI64	00-00-00-00-00-00-00-0		; Missing char in a hex pair
@	EUI64	00:00-00-00-00-00-00-00		; Bad separator
@	EUI64	00-00-00-x0-00-00-00-00		; Bad character
@	EUI64	00-00-00-00-00-00-00-00 x	; Unexpected item
@	EUI64	\# 9 0000000000000000 00	; Too long rdata
@	EUI64	\# 9 0000000000000000		; Bad rdata length
