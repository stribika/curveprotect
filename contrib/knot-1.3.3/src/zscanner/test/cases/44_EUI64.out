OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006D
RDATA=0000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006D
RDATA=FFFFFFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006D
RDATA=AABBCCDDEEFF0102
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006D
RDATA=0000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006D
RDATA=0000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006D
RDATA=0000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006D
RDATA=0000000000000000
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_EUI_LENGTH
------
WARNG=ZSCANNER_EBAD_EUI_LENGTH
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_CHAR_DASH
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
